test/mocks/stats/mocks.h

```diff
@@ -318,6 +318,9 @@ class MockIsolatedStatsStore : public SymbolTableProvider, public IsolatedStoreI
   ~MockIsolatedStatsStore() override;
 
   MOCK_METHOD2(deliverHistogramToSinks, void(const Histogram& histogram, uint64_t value));
+  MOCK_METHOD2(histogram, Histogram&(const std::string&, Histogram::Unit));
```

Comment:
Is this method used anywhere? I might just be missing something, but it's not clear to me why it's necessary.

Reply:
Remnants from trying to be more clever than the testing approach allowed from this class, removed.